### DISCUSSION TITLE
Links to anchors show the wrong link text in HTML output

### DISCUSSION THREAD

**Discussion by kgarmtz** [dita-ot/dita-ot#4158] [Q&A]
Hi team, good morning.
I have the following xref elements in my source document:

```
<xref keyref="GUID/ID"/>

<xref format="dita"
              keyref="GUID/ID"
              scope="local" type="section"/>
```

In the output (toc.htm) all the `xref`s that were defined with a `keyref` attribute are not getting the correct title, they show the title of the target topic, not the title of the section that is being linked to, though the links point to different anchors on the page.  It is important to notice that this behavior is not happening with`xref `s that were defined with an `href` attribute, those are getting the correct tittle.

I'm using the DITA-OT 1.8.5 version, so I want to know if there is any workaround for this problem?
Thanks in advance team.
Kind regards,
Kevin.

**Comment by raducoravu** [comment]
@kgarmtz the DITA OT 1.8.5 was released in 2014, about 10 years ago. Almost nobody uses it anymore, it's not officially maintained anymore. 
Do you also have extra publishing plugins installed in the DITA OT? This would complicate the situation because the problem could be either invalid DITA content or a bug in the DITA OT or a bug in the publishing plugins.
In general when you have any kind of publishing problem ideally you should try to come up with a very small sample project (DITA Map + one or two topics) to reproduce the problem, zip the project and attach it to the discussion. 
Once you have such a small sample project you can also try to publish it using the latest DITA OT distribution to see if it works better or not.

Linking also to the discussion you started on the Oxygen forum: https://www.oxygenxml.com/forum/post70368.html#p70368
